tests/test_browser.py

```diff
@@ -4721,3 +4721,6 @@ def test_minimal_runtime_loader_shell(self):
       for modularize in [[], ['-s', 'MODULARIZE=1']]:
         print(str(args + wasm + modularize))
         self.btest('minimal_hello.c', '0', args=args + wasm + modularize)
+
+  def test_boost_graph(self):
+    self.btest('test_boost_graph.cpp', expected='0', args=['-s', 'USE_BOOST_HEADERS=1'])
```

Comment:
this would be better in `other` rather than `browser`, since browser tests are much slower (using `do_other_test` there it's pretty simple too).

Reply:
okay